lib/tests/test_local_working_copy_sparse.rs

```diff
@@ -126,7 +125,7 @@ fn test_sparse_checkout() {
         repo.store().clone(),
         ws.workspace_root().to_path_buf(),
         wc.state_path().to_path_buf(),
-        &user_settings(),
+        repo.settings(),
```

Comment:
nit: should be squashed to the first commit

Reply:
Done.